cli/commands/clean_db/__init__.py

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (C) 2024 Red Hat, Inc.
```

Comment:
This function only removes data based one the configuration. It would only remove data for a specific team. Let's say OCM wants data for 90 days but OCP wants it for 180 or 365, this solutions allows the individual teams to decide without a single job limiting the entire organization to how far back they can look at bugs. 

The only reason I say this is that some team's dashboards may have more interest from leadership than others. If OpenShift QE on-boards, I imagine leadership may want to view an entire year's worth of data or at least more than 1 quarter worth of data. But there may be some teams where it only makes sense to view data for 90 days.

Reply:
That being said, if you want to limit everyone, that’s also something we can discuss in our meeting.

Reply:
I have come to realization that cleaning the db per team is not important as each JQL query can pull results for however many days a team would like to display. I will change this functionality today.